vite.config.ts

```diff
@@ -27,5 +27,15 @@ export default defineConfig({
       '@': path.resolve(__dirname, './src'),
       '@components': path.resolve(__dirname, './src/components')
     }
+  },
+  css: {
+    preprocessorOptions: {
+      scss: {
+        additionalData: '@import "@/static/styles/vars/_base.scss";'
+      },
+      less: {
+        additionalData: '@import "@/static/styles/vars/_base.less";'
+      }
+    }
```

Comment:
我本来的意思是给你的改动代码加注释

然后样式资源放到`src/styles`目录下不是`src/static/styles`，这里的值我改了，辛苦你改一下实际的文件 路径
```suggestion
  css: {
    // 配置`scss`和`less`全局变量
    preprocessorOptions: {
      scss: {
        additionalData: '@import "@/styles/vars/_base.scss";'
      },
      less: {
        additionalData: '@import "@/styles/vars/_base.less";'
      }
    }
```